### ISSUE TITLE
Show rate on transaction details page if a fee was subject to a currency exchange rate

### ISSUE THREAD

**Issue by jrodger** [Automattic/woocommerce-payments#1342]
The currency of the fee charged for a transaction is based on the store's location (to be precise, the location specified in the Stripe on-boarding when setting up the WooCommerce Payments account).

If a transaction is made in a different currency, we have to convert the fee into the same currency. This will use an exchange rate determined by WooCommerce Payments, which may be different to the exchange rate used by Stripe for the transaction itself.

To avoid confusion, all this information needs to be accessible in the transaction details page.

![image](https://user-images.githubusercontent.com/686419/108241485-cbbf6e00-7143-11eb-88f9-fafcab558da0.png)

Details: paJDYF-13A-p2

**Comment by vbelolapotkov** [comment]
Marking as blocked until its relevance is clarified with David.

**Comment by rogermattic** [comment]
hey @vbelolapotkov ! just wanted to check if there's been any update on this?

**Comment by vbelolapotkov** [comment]
@rogermattic, no progress I'm aware off. I see it's been relabeled for Gamma, so tagging @mjdeacon and @deepakpathania to provide any further updates.

**Comment by deepakpathania** [comment]
No updates from us, was added to our backlog based on the `positioning and fees` label but we haven't prioritised it yet. Is there any context that could help us bump it's priority? Otherwise we'd triage it normally as a part od our process next sprint.

**Comment by rogermattic** [comment]
> Is there any context that could help us bump it's priority?

Not at all! But I was wondering if I could help extracting and adding a solution here based on  my design explorations in this P2 post: p6riRB-9T6

**Comment by dmvrtx** [comment]
Thank you @rogermattic for the exploration and suggested designs. There was a missing piece of information about the fee conversion rate, which was not available for the timeline. It will be added on the server side.

**Comment by rogermattic** [comment]
Awesome! 

As mentioned on Slack I noticed the P2 and design reference was outdated here so I wanted to update it here:
- latest P2 on this: p6riRB-a8m
- latest [design in Figma](https://www.figma.com/design/NfXDn8fUFw0CHZutHitjmb/Displaying-Fees?node-id=759-51314&t=WKuJ5RYuWVdKiNND-1)
Let me know in case anything design-wise needs clarification etc.

Let's keep the conversation going here!

**Comment by allie500** [comment]
👋  Should we move this to the 9.2.0 milestone?

**Comment by dmvrtx** [comment]
> 👋 Should we move this to the 9.2.0 milestone?

Good point! Done ✅
